pkg/apis/db/v1alpha1/cassandracluster_types.go

```diff
@@ -567,6 +567,21 @@ func (cc *CassandraCluster) getDCFromIndex(dc int) *DC {
 	return &cc.Spec.Topology.DC[dc]
 }
 
+// Get DC by one of its rack name
+func (cc *CassandraCluster) GetDCByRackName(dcRackName string) *DC {
+	index := cc.GetDCIndexFromDCName(cc.GetDCFromDCRackName(dcRackName))
+	// Not found
+	if index == -1 {
+		return nil
+	}
```

Comment:
I don't think you need to check it as it's called internally and we know we have that dcRackName